assets/styles/common/purpose.scss

```diff
@@ -201,9 +201,16 @@
     text-decoration: none;
 
     .content-purpose-content {
+
       .title {
         font-size: 1.3em;
-        color: var(--color-body-color);
+        @media (prefers-color-scheme: light) {
+          color: var(--color-body-color);
+        }
+
+        @media (prefers-color-scheme: dark) {
```

Comment:
Instead of using @media selector, can we add this to the `html.dark` rules below? This will cover both cases where dark mode is system driven and JS preference driven (both add `dark` class to html element) We can also utilize the `--color-body-color-dark` css variable that get generated by tailwind. e.g.
```
html.dark {
  ...

  .content-purpose-content {
    .title {
      color: var(--color-body-color-dark)
    }
  }

```